Gds/README.md

```diff
@@ -7,42 +7,42 @@ for instructions on how to use the GDS.
 The GDS consists of a collection of classes and tools that provide an interface for fprime
 deployments, allowing users to view telemetry and events and send commands.
 
-The GDS HTML GUI is an almost completely rewritten version of the fprime GSE UI, our historical
+The GDS HTML GUI is an almost completely rewritten version of the F' GSE UI, our historical
```

Comment:
Did you mean:

```suggestion
The GDS HTML GUI is an almost completely rewritten version of the F´ GSE UI, our historical
```

You used this notation near the bottom...

Reply:
Yes, we should use F´ or F Prime in all cases.

Reply:
My apologizes, didn't even realize I had used that character.. See latest commit for the notation fix.

https://github.com/nasa/fprime/pull/527/commits/d6399779c3b7e7a3872429c727c766bb74222ff2

Reply:
It is a subtle distinction, thanks for improving this!  We'll let CI run, and since there are no code changes I'll merge when CI passes.

Reply:
I didn't know which was correct. I randomly read PRs here. I tend to ask about consistency because it's an easy contribution without requiring much technical knowledge on my part.

Reply:
More than fine.  We appreciate your contributions!